book/src/transforms/select.md

```diff
@@ -35,6 +37,37 @@ from e=employees
 select [e.first_name, e.last_name]
 ```
 
+### Excluding columns
+
+We can use `!` to exclude a list of columns. This can operate in two ways:
+
+- We use `SELECT * EXCLUDE` or `SELECT * EXCEPT` for the supplied columns in
+  dialects which support it.
+- Otherwise, the columns much have been defined prior in the query (unless all
+  of a table's columns are excluded); for example in another `select` or a
+  `group` transform.
```

Comment:
```suggestion
We can use `!` to exclude a list of columns.

Note that in some cases, the columns will actually be excluded in the output SQL. This is because it is not possible to express such queries in SQL, due to limited support for `SELECT * EXCLUDE/EXCEPT` in many dialects. We can workaround these cases by selecting all of the columns of an input table prior in the pipeline.
```

Reply:
Thanks, added a modified version of this to the bullets; feel free to adjust further